backend/ventserver/simulation/power_management.py

```diff
@@ -33,6 +33,32 @@ def transform(
         else:
             self._transform_discharge(power_management, events_log)
 
+        if power_management.power_left <= 30:
+            events_log.input(alarms.AlarmActivationEvent(
+                code=mcu_pb.LogEventCode.battery_low,
+                event_type=mcu_pb.LogEventType.system
+            ))
+        else:
+            events_log.input(
+                alarms.AlarmDeactivationEvent(
+                    codes=[mcu_pb.LogEventCode.battery_low]
+            ))
+
+        if power_management.power_left <= 5:
```

Comment:
When `power_left <-==5`, it looks like  the backend simulator is going to activate and then deactivate the `battery_low` alarm. This means that if the simulator starts up with critical battery levels, then the firmware will create a spurious `battery_low` log event.

Here are two ways you could implement this differently:
1. First do your if/else case handling to convert your numerical `power_left` into an enum representing power level bins (normal, low, critical). Afterwards, do if/else case handling to convert the power level bin to alarm activation/deactivation. This would be more similar to what the firmware does.
2. Figure out the correct if/else case handling to directly convert `power_left` into alarm activation/deactivation. For example, this could look like `if power_left <= 5, do something, else if power_left <= 30, do something else, else do something else`.